src/plugins/dashboard/public/dashboard_app/_dashboard_app.scss

```
@@ -1,3 +1,13 @@
+@import 'src/core/public/mixins';
+
+.dshAppWrapper {
+  @include euiBreakpoint('xs', 's', 'm', 'l', 'xl') {
+    @include kibanaFullBodyHeight();
+  }
```

Since this is now using all available breakpoints, we should be able to remove the media query altogether.

```suggestion
  @include kibanaFullBodyHeight();
```